provider/k8s.go

```diff
@@ -300,7 +301,7 @@ type KubernetesExecutor struct {
 func (kube *KubernetesExecutor) isDefaultImage() (bool, error) {
 	parse, err := dp.Parse(kube.image)
 	if err != nil {
-		return false, fmt.Errorf("invalid image name: %w", err)
+		return false, fferr.NewInvalidArgument(err)
```

Comment:
We may want to prepend "invalid Docker image name: err" to this, unsure what error that library is going to throw